Backpack/Dialog/README.md

```diff
@@ -0,0 +1,68 @@
+# Backpack/Dialog
+
+## Installation
+
+In `Podfile` add
+
+```
+pod 'Backpack/Dialog'
+```
+
+and then run `pod install`.
+
+## Usage
+
+`Backpack/Dialog` contains the Backpack Dialog component in the class `BPKDialogController` .
+
+
+### Objective-C
+```
+#import <Backpack/Dialog.h>
+
+...
+
+BPKDialogScrimAction *scrimAction = [BPKDialogScrimAction actionWithHandler:^(BOOL didDismiss) {
+    //
+} shouldDismiss:YES];
+
+BPKDialogButtonAction *buttonAction = [BPKDialogButtonAction actionWithTitle:@"Continue" style:BPKButtonStylePrimary handler:^{
+    //
+}];
+
+BPKDialogController *dialogController = [BPKDialogController dialogControllerWithTitle:@"Such Wow!"
```

Comment:
We should probably update the readme example to use the copy we applied to the stories

Reply:
Sounds like a good idea yeah